test/integration/store_admin_receives_request_when_order_placed_test.rb

```diff
@@ -0,0 +1,66 @@
+require "test_helper"
+
+class StoreAdminReceivesRequestWhenOrderPlacedTest < ActionDispatch::IntegrationTest
+  def login_store_admin
+    Store.first.update(approved: true)
+    User.find_by(username: "Dave").roles.create(name: 'store_admin')
+
+    visit login_path
+    fill_in 'Username', with: 'Dave'
+    fill_in 'Password', with: 'cars'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.